Orange/widgets/regression/owrandomforestregression.py

```diff
@@ -1,35 +1,24 @@
-from PyQt4 import QtGui
-from Orange.data import Table
-from Orange.regression.random_forest import (RandomForestRegressionLearner,
-                                             RandomForestRegressor)
-from Orange.widgets.classify.owrandomforest import OWRandomForest
+from Orange.regression.random_forest import RandomForestRegressionLearner
+from Orange.widgets.classify import owrandomforest
 from Orange.widgets import settings
 
 
-class OWRandomForestRegression(OWRandomForest):
+class OWRandomForestRegression(owrandomforest.OWRandomForest):
     name = "Random Forest Regression"
     description = "Random forest regression algorithm."
+    icon = "icons/RandomForest.svg"
+    priority = 40
 
     LEARNER = RandomForestRegressionLearner
-    outputs = [("Learner", LEARNER),
-               ("Model", RandomForestRegressor)]
+    OUTPUT_MODEL_NAME = "Model"
```

Comment:
I can give you my current preference, but I'm known to occasionally change my mind, especially w.r.t. terminology. :)

*Model* can (often) be used as *predictor*. Model is a result of modelling; like in *supervised and unsupervised modelling*. For instance, classifiers and regressors are called *predictive models*. Clustering is also a model (of the data), although typically not a predictor.

*Model* refers to content ("model describes the data"), while *Predictor* describes the use ("predictor classifies the data instances").

From this point on, I leave the decision to others. :) I don't care so much.